sorbet/config

```diff
@@ -1,6 +1,7 @@
 --dir
 .
 --ignore=vendor/
+--ignore=tmp/
```

Comment:
I'd rather keep the ignore in case the indexing fails part way through, and leaves some files in there. The machine grinds to a halt if tries to start typechecking those gems directories.